src/shrutiS/assignment18/MaxFrequencyCharacter.java

```diff
@@ -0,0 +1,36 @@
+/*Program : 1
+max repeating word with its freq.
+input : Hi Hello Hi Techno Hello Hi
+output : Hi -> 3*/
+package shrutiS.assignment18;
+
+public class MaxFrequencyCharacter {
+	int maxFrequency;
+	String word;
+	void maxFrequencyCharCount(String input) {
+		String[] arr = input.split(" ");
+		for (int index = 0; index < arr.length; index++) {
+		    word = arr[index];
+			int count = 0;
+			if (input.contains(word)) {
+				for (int innerIndex = 0; innerIndex < arr.length; innerIndex++) {
+					if (word.equals(arr[innerIndex])) {
```

Comment:
Here you already apply .equals() method then no need to use contains() method, think here, you are checking twice for same word, & you not putted else condition for contains()  method condition. 

here you should remove this condition.